magclassic/automated_emails.py

```diff
@@ -1,19 +1,25 @@
 from magclassic import *
 
-# With "MAGFest Laboratories" as a new event, we find that the phrasing "Staffing again"
-# doesn't differentiate this event, so we remove it.
-AutomatedEmail.instances['Want to staff {}?'.format(c.EVENT_NAME)] = AutomatedEmail.instances.pop('Want to staff {} again?'.format(c.EVENT_NAME))
+# With "MAGFest Laboratories" as a new event, we find that the phrasing "Want to staff again?"
+# doesn't differentiate this event, so we change it to just plain "Want to staff?"
+if 'volunteer_again_inquiry' in AutomatedEmail.instances:
```

Comment:
Okay, that makes perfect sense! I deleted those lines, and committed straight to master (because it was really a follow-on to this prematurely merged pull request, I did reference this PR, tho).

I had assumed everyone on the east coast had quit working!